Ira/my-app-video/src/components/layout/header/Header.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {Logo} from '../../shared/logo/Logo';
+import {Button} from "../../shared/button/Button";
+import {FullInfoFilmCard} from "../../fullInfoFilmCard";
+import {IHeaderProps} from "../../../interfaces/IHeaderProps";
+import './styles.scss';
+
+export const Header = ({
```

Comment:
It doesn't seem like Header. Header should contain logo and nav link. Better to move part of this component into a separate one (move film info and form into separate components ).